helmfile/charts/gatekeeper/constraints/templates/reject-loadbalancer-service/constraint.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.rejectLoadBalancerService.enable -}}
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRejectLoadBalancerService
+metadata:
+  name: elastisys-reject-load-balancer-service
+spec:
+  enforcementAction: {{ .Values.rejectLoadBalancerService.enforcementAction }}
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Service"]
+    namespaceSelector:
+      matchExpressions:
+      - key: owner
+        operator: NotIn
+        values:
+        - operator
```

Comment:
I wonder if we should perhaps skip this namespace filter and just apply this to all namespaces. Because this policy will be enabled if we cannot use loadbalancer services and then it is a problem in all namespaces. What do you think about that idea?

Reply:
What if we can create only one one load balancer that we use for the ingress-controller?

Reply:
Yeah that is a good reason to keep it or possibly have a new config option to toggle this.

Reply:
So, keep as-is? @aarnq @viktor-f